src/Nowruz/modules/Auth/containers/onboarding/OrganizationContact/useOrganizationContact.tsx

```diff
@@ -73,12 +77,16 @@ export const useOrganizationContact = () => {
       );
       localStorage.removeItem('registerFor');
       reset();
-      if (isMobile)
+      dispatch(setIdentityList(new_identities));
+      if (isMobile) {
+        dispatch(setIdentityList(new_identities));
```

Comment:
I think we don't need the one inside the condition as we already have on prior the condition?